src/components/Container/Container.jsx

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+
+const boundClassNames = lucidClassNames.bind('&-Container');
+
+const {
+	node,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"]}
+ *
+ * Container is used to wrap content to better organize elements in window.
+ */
+const Container = React.createClass(createLucidComponentDefinition({
+	displayName: 'Container',
+
+	childProps: {
+		Header: null,
+		Content: null,
```

Comment:
We don't need a `Content` child prop, it's simply anything that isn't a `Header` or `Footer`